flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBIncrementalCheckpointUtils.java

```diff
@@ -138,7 +138,7 @@ public static void clipDBWithKeyGroupRange(
      * @param beginKeyBytes the begin key bytes
      * @param endKeyBytes the end key bytes
      */
-    private static void deleteRange(
+    public static void deleteRange(
```

Comment:
We can make the scope of this method package-private.

Reply:
Resolved, please help to review it again.